### DISCUSSION TITLE
Keep a history of the screens I have looked at and allow me to quickly move back to a previous screen without having to search for it.

### DISCUSSION THREAD

**Discussion by Uggi** [microsoft/BotFramework-Composer#5795] [General]
Please provide the equivalent of visual studios "Navigate forward/back".
![image](https://user-images.githubusercontent.com/4349731/108003339-69068f00-6fea-11eb-98ae-7fb0d33aea62.png)

Navigation in Composer is painful when you have a lot of dialogs & bot composer & lg files etc. 
The moment you move away from a page, regardless of type (dialog/bot response/qna etc) you lose all context of where you have been. There isn't even a back button! You then have to hunt around in whichever area you came from to find the relevant section. 

this is really very obvious if you have a shared dialogs that are called from multiple places and you are following a path through them. there is no way to navigate back.

**Comment by a-b-r-o-w-n** [comment]
@Uggi Are you using the desktop version of Composer?

**Comment by a-b-r-o-w-n** [comment]
@Uggi Please open a feature-request issue and we will put this in our backlog.
